### PR TITLE
Basic command setup

### PR DESCRIPTION
**Pull request by MrDienns**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -47,6 +45,25 @@
...
+        <dependency>
+            <groupId>co.aikar</groupId>
+            <artifactId>acf-jda</artifactId>
+            <version>0.5.0-SNAPSHOT</version>
+        </dependency>
+
+        <dependency>
+            <groupId>joda-time</groupId>
+            <artifactId>joda-time</artifactId>
+            <version>2.10.1</version>
```
Comment (MiniDigger) [inline pom.xml]: Why was this added and why would anyone want to use joda-time in 2019 when the creator wrote the java.time API?

**Review by MiniDigger — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (MrDienns) [reply pom.xml]: I need to be able to turn values like `5h10m30s` into actual time objects... Googling around it resulted in joda-time. I guess I can achieve the same without it then?

Comment (MiniDigger) [reply pom.xml]: java.time.Period/Duration.parse("PT5h10m30s")
it accepts the iso-idk format for periods.

**Review by MiniDigger — APPROVED** [review]
